lib/fluent/plugin/parser_json.rb

```diff
@@ -30,6 +30,12 @@ class JSONParser < Parser
       desc 'Set JSON parser'
       config_param :json_parser, :enum, list: [:oj, :yajl, :json], default: :oj
 
+      # The Yajl library defines a default buffer size of 8092 when parsing
```

Comment:
[nits] It may be better a default buffer size is 8192  because of https://github.com/brianmario/yajl-ruby/commit/9d4cca0175142c2308746d7c1d40d89b682b168f

Reply:
FYI: https://github.com/brianmario/yajl-ruby/pull/192

Reply:
Definitely not a nit; thanks for picking up on this! I did think that 8092 bytes was a bit of a strange size for a buffer, but didn't look into it 🤦‍♂️. 
Fixed and I'll update the docs PR too.